### DISCUSSION TITLE
React Markdown showing (`) backquote/backtick on render.

### DISCUSSION THREAD

**Discussion by rifqirosyidi** [remarkjs/remark#674] [Q&A] [answered]
Hi there,

I just wondering if the default display for **code** backtick in markdown i think it  should remove the backtick by default but they still showing on my page. here's what is look on rendered page

screenshot:
![Screenshot from 2021-03-29 19-30-53](https://user-images.githubusercontent.com/41833714/112836964-77ee6000-90c5-11eb-8f23-a15202a110ac.png)

i also try to inspect html there are ::before and ::after that render those backtick. is there any way i can remove this. and also i use tailwind-typograpy. is this the probem with tailwind or just default react-markdown?

**Comment by wooorm** [comment, marked as answer]
there are no backticks added with `before` and `after` by this project. You should look for the problem somewhere else

**Reply by wooorm** [reply]
https://github.com/tailwindlabs/tailwindcss-typography/blob/master/src/styles.js#L116-L121

**Comment by peteristhegreat** [comment]
The fix in tailwind typography that adds this is to put an patch in the `tailwind.config.js` file

The default values for ::before and ::after look like

```
        'code::before': {
          content: '"`"',
        },
        'code::after': {
          content: '"`"',
        },
```

This is one possibility for changing it:

```
module.exports = {
  mode: 'jit',
  content: [
    './src/pages/**/*.{js,ts,jsx,tsx}',
    './src/components/**/*.{js,ts,jsx,tsx}',
  ],
  theme: {
    extend: {
      typography: {
        DEFAULT: {
          css: {
            'code::before': {
              content: '&nbsp;&nbsp;',
            },
            'code::after': {
              content: '&nbsp;&nbsp;',
            },
            code: {
              background: '#ffeff0',
              wordWrap: 'break-word',
              boxDecorationBreak: 'clone',
              padding: '.1rem .3rem .2rem',
              borderRadius: '.2rem',
            }
          },
        },
      },
    },
  },
  plugins: [require('@tailwindcss/typography')],
};
```

**Reply by traversng** [reply]
Thanks bud 🤙🏽

**Comment by maheshj01** [comment]
With the above fix the build fails with error
```
HookWebpackError: /vercel/path0/static/css/92438371fc38e653.css:717:18: Unknown word
```
Use this instead

```js
    typography: {
      DEFAULT: {
        css: {
          'code::before': {
            content: '""',
          },
          'code::after': {
            content: '""',
          },
        },
      },
    },
```
